services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2337,6 +2337,15 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
           && clusterConfig.isActiveActiveReplicationEnabledAsDefaultForHybrid()) {
         setStore.activeActiveReplicationEnabled = true;
         updatedConfigsList.add(ACTIVE_ACTIVE_REPLICATION_ENABLED);
+        if (!hybridDataReplicationPolicy.isPresent()) {
+          LOGGER.info(
+              "Data replication policy was not explicitly set when converting store to hybrid store: {}."
```

Comment:
Can we also update `VeniceHelixAdmin.mergeNewSettingsIntoOldHybridStoreConfig` with a default DRP? So, `VeniceHelixAdmin` can set it to `NON_AGGREGATE`, and `VeniceParentHelixAdmin` can set it to `ACTIVE_ACTIVE` or `NON_AGGREGATE` based on the AA configs. Otherwise, every hybrid store creation will log this line

Reply:
Logging is intentional here as we want to change policy to AA if and only store is being converted to hybrid mode and user did not provide any policy when converting store to hybrid. 

I think this is much better place to override the policy as it doesn't required duplication of checks in  `VHA::mergeNewSettingsIntoOldHybridStoreConfig` and also keeps the hybrid store conversion logic grouped together

Reply:
Makes sense. LGTM